tools/utils.cc

```diff
@@ -15,6 +15,10 @@
 
 namespace tools::utils {
 
+bool operation::matches(std::string_view name) const {
+    return _name == name || std::ranges::find(_aliases, name) != _aliases.end();
```

Comment:
std::ranges::contains

Reply:
```
 build/dev/tools/utils.o tools/utils.cc
tools/utils.cc:19:42: fatal error: no member named 'contains' in namespace 'std::ranges'
    return _name == name || std::ranges::contains(_aliases, name);
                            ~~~~~~~~~~~~~^
1 error generated.
ninja: build stopped: subcommand failed.
```

Reply:
yes, I (now) remember making the same comment to @tchaikov and receiving a similar response.